assessment.sql

```diff
@@ -5,24 +5,35 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+ id SERIAL PRIMARY KEY,
+ surname CHARACTER VARYING,
+ given_name CHARACTER VARYING,
+ gender CHARACTER VARYING(1),
+ height INTEGER,
+ weight INTEGER,
+ dob DATE,
+ eye_color CHARACTER VARYING,
+ hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO(surname,given_name,gender,height,weight,dob,eye_color,hair_color);
+VALUES('Bishop','Caren,','f', 63 ,132 , 1943-09-26, 'Brown', 'Black');
```

Comment:
This will fail for a cryptic reason. It will claim you tried to insert an integer into a date column. In fact, `1943-09-26` can't be distinguished from `1943` minus `09` minus `26`, or `1908`. For that reason, dates (frequently called **datestrings**) are specified as strings, like the following: `'1943-09-26'`. 3/4